pandas/core/frame.py

```diff
@@ -2967,6 +2967,12 @@ def query(self, expr, inplace=False, **kwargs):
             The query string to evaluate.  You can refer to variables
             in the environment by prefixing them with an '@' character like
             ``@a + b``.
+
+            .. versionadded:: 0.25.0
+
+            You can refer to column names that contain spaces by surrounding
+            them in backticks like ```a a` + b``.
```

Comment:
can you expand this comment, its hard to interpret what a user should do here

Reply:
Made the example more explicit.